lib/matplotlib/artist.py

```diff
@@ -575,9 +802,14 @@ def get_sketch_params(self):
 
         May return `None` if no sketch parameters were set.
         """
-        return self._sketch
+        msg = ("This has been deprecated to make way for IPython's"
```

Comment:
Maybe remove the `IPython's` here (and elsewhere similar messages appear).  This is not to avoid giving credit where credit is due, just that this could imply that we are growing a dependency on IPython, which we aren't.